src/ApolloHooksTypes.re

```diff
@@ -43,18 +44,4 @@ let fetchPolicyToJs = fetchPolicy =>
   | NoCache => "no-cache"
   | Standby => "standby"
   };
-
-/**
- * apollo-client/src/core/watchQueryOptions.ts
- */
-type errorPolicy =
```

Comment:
Are those not used anymore?

Reply:
Ah that is probably an upstream change that I forgot to merge in (diffs were not working anymore so I had to manually look up what has been added to master in the meantime)